### PR TITLE
Add a simple FAQ

### PR DESCRIPTION
**Pull request by joshbressers**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# SBOM Everywhere Catalog FAQ
+
+## What is the SBOM Catalog?
+The OpenSSF has a working group called “SBOM Everywhere”, one of the projects this group is working on is a place to catalog as much SBOM related information as possible. Because the OpenSSF has a focus on open source, we’re doing it as an open source project! The goal is to capture anything that’s SBOM related and make it easy to find. Projects, documents, standards, working groups, anything that could be useful.
```
Comment (sjn) [inline FAQ.md:4]: add link to SBOM Everywhere project page, maybe?

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Who can update this?
+Anyone can contribute to this. It’s open source! You can see all the details in our GitHub repo
+https://github.com/ossf/sbom-everywhere
+
+## What sort of information can we add?
+We are trying to capture and track as much information about the SBOM ecosystem we can. Meetings, projects, tools, formats, standards, everything. If you’re unsure, let’s chat about it, you can find our details here https://sbom-catalog.openssf.org/about-us.html
+
+## How can I add an entry or fix an error to the Catalog or Wiki?
+Open an issue in GitHub https://github.com/ossf/sbom-everywhere/issues
```
Comment (sjn) [inline FAQ.md:19]: _Strictly speaking_, an issue tracker _isn't_ a way to edit a wiki. How about linking to the place where the wiki can be edited? Or don't call it a wiki. :sweat_smile:

**Review by sjn — COMMENTED** [review]
Looks good to me! Two nits to pick, but nothing major.

**Comment by funnelfiasco** [comment]
I agree with sjn's comments, but otherwise this looks good. I wouldn't object to merging as-is and making refinements later, either.

File: FAQ.md (hunk 2 above)
Comment (joshbressers) [reply FAQ.md:19]: I'm OK calling it a wiki. We edit it via PRs in this instance, but we don't have the PR instructions ready yet

**Review by funnelfiasco — APPROVED** [review]
Looks good to me. (I don't have write access on this repo, so my review doesn't "count", but...)

**Pull request merged**